internal/cli/cli.go

```diff
@@ -125,6 +131,10 @@ func (c *RootCommand) runE(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	if err := v1beta1controllers.NewGarbageCollectionController(certmanagerClient, ic).WithDryRun(dryRun).SetupWithManager(ctx, mgr); err != nil {
```

Comment:
Since the `GarbageCollectionController` `WithDryRun` will clash with existing `GatewayController` `WithDryRun`, I moved the GarbageCollectionController to its own package within `internal/controllers/v1beta1`.

Let me know if I should move the `Gateway` controller to its own package/dir as well.

Reply:
These should probably be their own controller packages within v1beta1.  This is good.